packages/embark/src/lib/utils/escapeHtml.js

```diff
@@ -0,0 +1,12 @@
+function escapeHtml(message) {
+  if(typeof message !== "string") return message;
+
+  return message
+    .replace(/&/g, "&amp;")
```

Comment:
Is there a library that would handle this for us and possibly handle additional use cases that we haven't thought of?

Reply:
Probably tons of them, but this is a simple enough fix that I think we can handle it ourselves without introducing yet another dependency.